src/renderer/library/components/dialog/ApiappAddForm.tsx

```diff
@@ -120,13 +122,26 @@ const ApiappAddForm = () => {
                                 e.stopPropagation();
                                 e.preventDefault();
                                 const v = searchInputRef.current?.value;
-                                if (!v) return ;
+                                if (!v) return;
                                 apiAppSearchAction(v);
                                 // e.preventDefault();
                                 // e.stopPropagation();
                             }
                         }}
                     />
+                    {searchInputRef ?
```

Comment:
The ref itself is always truthy ... shouldn't this be:
`{ searchInputRef.current ?`